src/ert/shared/hook_implementations/workflows/export_runpath.py

```diff
@@ -28,9 +33,13 @@ class ExportRunpathJob(ErtScript):
     file.
     """
 
-    def run(self, *args: str) -> None:
-        _args = " ".join(args).split()  # Make sure args is a list of words
-        config = self.ert().ert_config
+    def run(
+        self, ert_config: ErtConfig, input_ranges: Optional[List[str]] = None
+    ) -> None:
+        input_ranges = [] if input_ranges is None else input_ranges
+        _args = " ".join(input_ranges).split()  # Make sure args is a list of words
+        config = ert_config
```

Comment:
I think this warrants a rewrite of the use of config instead of just assigning `ert_config` to `config`.